### DISCUSSION TITLE
Custom ComfyUI workflow

### DISCUSSION THREAD

**Discussion by kubilaykilinc** [Acly/krita-ai-diffusion#721] [General]
Is there a way to use Custom ComfyUI workflow with Krita and Krita Diffusion?

**Comment by sraney** [comment]
I was about to ask this, I would like to see one or more of the the background removal workflows added.

**Comment by piotrkucharski** [comment]
There was a discussion around this idea:

https://github.com/Acly/krita-ai-diffusion/discussions/385
